README.md

```diff
@@ -183,87 +183,587 @@ raw index but it will enriched in the enriched index.
+            "https:/github.com/chaoss/grimoirelab-sirmordred"
+        ]
+    }
+}
+```
+- setup.cfg
+```
+[github2:pull]
+api-token = xxxx
+raw_index = github2-pull_raw
+enriched_index = github2-pull_enriched
+sleep-for-rate = true (suggested)
+no-archive = true (suggested)
+category = pull_request
+```
 
 #### gitlab
+Issues and MRs from GitLab
 
 GitLab issues and merge requests need to be configured in two different sections.
+And don't forget to enable the `gitlab-issue` and/or `gitlab-merge` in the `pnales` section in `setup.cfg`. 
 
+If a given GitLab repository is under more than 1 level, all the slashes `/` starting from the second level have to be
+replaced by `%2F`. For instance, for a repository with a structure similar to this one
+`https://gitlab.com/Molly/lab/first`.
+##### issue
+- projects.json
 ```
 {
     "Chaoss": {
         "gitlab:issues": [
             "https://gitlab.com/Molly/first",
-            "https://gitlab.com/Molly/second"
-        ],
+            "https://gitlab.com/Molly/lab%2Fsecond"
+        ]
+    }
+}
+```
+- setup.cfg
+```
+[panels]
+gitlab-issues = true
+
+[gitlab:issue]
+category = issue
+raw_index = gitlab-issues_raw
+enriched_index = gitlab-issues_enriched
+api-token = xxxx
+sleep-for-rate = true (suggested)
+no-archive = true (suggested)
+studies = [enrich_onion:gitlab-issue] (optional)
+
+[enrich_onion:gitlab-issue] (optional)
+in_index = gitlab-issues_enriched
+out_index = gitlab-issues-onion_enriched
+data_source = gitlab-issues
+```
+##### merge request
+- projects.json
+```
+{
+    "Chaoss": {
         "gitlab:merge": [
             "https://gitlab.com/Molly/first",
-            "https://gitlab.com/Molly/second"
+            "https://gitlab.com/Molly/lab%2Fsecond"
         ],
     }
 }
 ```
+- setup.cfg
+```
+[panels]
+gitlab-merges = true
+
+[gitlab:merge]
+category = merge_request
+raw_index = gitlab-mrs_raw
+enriched_index = gitlab-mrs_enriched
+api-token = xxxx
+sleep-for-rate = true (suggested)
+no-archive = true (suggested)
+studies = [enrich_onion:gitlab-merge] (optional)
+
+[enrich_onion:gitlab-merge] (optional)
+in_index = gitlab-mrs_enriched
+out_index = gitlab-mrs-onion_enriched
+data_source = gitlab-merges
 
-If a given GitLab repository is under more than 1 level, all the slashes `/` starting from the second level have to be
-replaced by `%2F`. For instance, for a repository with a structure similar to this one
-`https://gitlab.com/Molly/lab/first`.
+```
 
+#### google_hits
+Number of hits for a set of keywords from Google
+- projects.json
 ```
 {
     "Chaoss": {
-        "gitlab:issues": [
-            "https://gitlab.com/Molly/lab%2Ffirst"
-        ],
-        "gitlab:merge": [
-            "https://gitlab.com/Molly/lab%2Ffirst"
-        ],
+        "google_hits": [
+            "bitergia grimoirelab"
+        ]
     }
 }
 ```
+- setup.cfg
+```
+[google_hits]
+raw_index = google_hits_raw
+enriched_index =google_hits_enriched
+```
+#### groupsio
```

Comment:
It would be useful to add a short sentence about how to get subscriptions
https://gist.github.com/valeriocos/ad33a0b9b2d13a8336230c8c59df3c55